.circleci/config.yml

```diff
@@ -4,11 +4,11 @@ executors:
     environment:
       IMAGE_NAME: suldlss/workflow-server
     docker:
-    - image: circleci/buildpack-deps:stretch
+    - image: cimg/buildpack-deps:stretch
```

Comment:
I don't think this image exists.

Reply:
You're right, thanks. Will revert that for now.